src/Particular.LicensingComponent/MonitoringThroughput/MonitoringThroughputHostedService.cs

```diff
@@ -0,0 +1,34 @@
+namespace Particular.LicensingComponent.MonitoringThroughput;
+
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using NServiceBus.Transport;
+using ServiceControl.Transports;
+using Shared;
+
+internal class MonitoringThroughputHostedService(ITransportCustomization transportCustomization, TransportSettings transportSettings, ILogger<MonitoringThroughputHostedService> logger, MonitoringService monitoringService) : IHostedService
+{
+    private TransportInfrastructure? transportInfrastructure;
+
+    private async Task Handle(MessageContext message, CancellationToken cancellationToken)
```

Comment:
It doesn't work - this never gets hit - messages are not being processed from the queue.

Reply:
I fixed that, we need to explicitly start the receiver